CHANGELOG.md

```diff
@@ -1,4 +1,35 @@
-## 4.6.0 (Unreleased)
+## 4.6.0 (January 10, 2021)
+
+BREAKING CHANGES:
+* pubsub: changed `google_pubsub_schema` so that modifiying fields will recreate the resource rather than causing Terraform to report it would attempt an invalid update ([#10768](https://github.com/hashicorp/terraform-provider-google/pull/10768))
+
+FEATURES:
+* **New Data Source:** google_container_aws_versions ([#10754](https://github.com/hashicorp/terraform-provider-google/pull/10754))
+* **New Data Source:** google_container_azure_versions ([#10754](https://github.com/hashicorp/terraform-provider-google/pull/10754))
+* **New Resource:** `google_apigee_nat_address` ([#10789](https://github.com/hashicorp/terraform-provider-google/pull/10789))
```

Comment:
do we want to make the resource names here uniform with the others?

Reply:
yep, looks like they're all supposed to have backticks! thanks